src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -6554,6 +6557,8 @@ namespace HeatBalanceSurfaceManager {
                 }
             } // ...end of loop to check for convergence
 
+            MaxAllowedDelTemp = DataConvergParams::MaxAllowedDelTempOverrideValue; 
```

Comment:
@jcyuan2020 Let's not reset this every time `CalcHeatBalanceInsideSurf2` is called. In my other branch, I'm going to move `MaxAllowedDelTemp` up into `DataConvergParams` so performance override can act on it directly.  This will create a conflict depending on which PR merges first.

Reply:
@mjwitte Thank you for the comments! We should definitely synchronize on this. I had originally considered a couple of options to make the variable `MaxAllowedDelTemp` global. One is to move it up to `EnergyPlus::DataConvergParams` as you indicated. Anyway, this called for a need to `#include <EnergyPlus/DataConvergParams.hh>` in `HeatBalanceSurfaceManager.cc` (as the current addition did in Line 130 in HeatBalanceSurfaceManager):
```
#include <EnergyPlus/DataConvergParams.hh>
```

The other neater way is to move `MaxAllowedDelTemp` into `EnergyPlus::DataHeatBalance`. This way could be much neater because: 1) it does not need the extra `#include <EnergyPlus/DataConvergParams.hh>` to be aded into `HeatBalanceSurfaceManager.cc`; 2) also, the `EnergyPlus/DataHeatBalance.hh` header file is already included in present `SimulationManager.cc`, so the Override mode can operate on this variable directly in `SimulationManager.cc` if `MaxAllowedDelTem` is in the `DataHeatBalance` namespace; 3) further, by putting `MaxAllowedDelTemp` into `EnergyPlus::DataHeatBalance`, it is actually then dealt by the same way as the `MaxAllowedDelTempCondFD` variable, which was already in `DataHeatBalance` prior to any changes (See the following code in Line 6562-6565 in `HeatBalanceSurfaceManager.cc`): 
```
if (!DataHeatBalance::AnyCondFD) {
                if (MaxDelTemp <= MaxAllowedDelTemp) Converged = true;
            } else {
                if (MaxDelTemp <= MaxAllowedDelTempCondFD) Converged = true;
```
What do you think? If you think the latter way can also work in your branch, we can synchronize this part by trying this option out as well.

Reply:
OK, I'll put it in DataHeatBalance.

Reply:
@mjwitte  The latter method discussed above is now implemented and committed. New commits were pushed, with old test unit fixed as well.